tests/Microsoft.Identity.Test.Unit/TestBase.cs

```diff
@@ -48,6 +60,15 @@ public virtual void TestInitialize()
         [TestCleanup]
         public virtual void TestCleanup()
         {
+            if (ManagedIdentityEnabled)
+            {
+                ManagedIdentityClient.s_sourceName = ManagedIdentitySource.None;
```

Comment:
There already exists a `TestCommon.ResetInternalStaticCaches()` which should do this all the time.

Reply:
removed in refactor